src/ASM/FiniteElement.C

```diff
@@ -109,7 +107,7 @@ bool MxFiniteElement::Jacobian (Matrix& Jac, const Matrix& Xnod,
 bool MxFiniteElement::Jacobian (Matrix& Jac, Vec3& n,
                                 const Matrix& Xnod,
                                 unsigned short int gBasis,
-                                const std::vector<Matrix>& dNxdu,
+                                const BasisValuesPtrs& dNxdu,
```

Comment:
Should maybe change the name of this parameter to `bfs` or `bf` like elsewhere, for consistency? Since this object also holds the basis function values themselves - not only the (first-order) derivatives

Reply:
yeah. i did that elsewhere but i guess i forgot to update this branch.